src/api-reference/common/extracts/v1.extracts.markdown

```diff
@@ -63,7 +69,7 @@ None
 
 #### Payload
 
-Name | Type | Format | Description
+Name|Type|Format|Description
 -----|------|--------|------------
```

Comment:
```suggestion
---|---|---|---
```

This is the most minimal amount of Markdown to specify a table + columns. If we are going to fix whitespace issues might as well get all of the tables to conform to the documentation template.

Reply:
Fixed